.ci-orchestrator/steps/fats/zOSUnittests.yml

```diff
@@ -0,0 +1,29 @@
+- stepName: z/OS Unittests
+  conditionalRun:
+    - type: ifTrue
+      value: ${spawn.zos}
+  workType: FAT
+  timeoutInMinutes: 1920
+  properties:
+    spawn.zos: true
+    ebcPlan: See Shortlist 
+    ebcShortlist: managed-pool-zos-fat-test-middleware.yml
+    reportingOS: zOS
+
+    # Product to be tested.
+    product_image_type_under_test: wlp-zosimage
+
+    # Buckets to be tested.
+    fat.buckets.to.run: ${PR Changes:fat.buckets.to.run}
+    fat.test.mode: lite
+    # Pattern causes us to only execute the z/OS unittests.
+    fatPatternToMatch: .*_ztest
+    runZosTests: true
+    command: ant -f build-zunittest.xml unittest -propertyfile ../../../buildandbucket.properties -DhaltOnFailure=false -lib ../ant_build/lib/ecj-4.3.1.jar -lib ../ant_build/lib.antClasspath -lib ../prereq.published/lib -lib ../ant_build/lib/biz.aQute.bnd-3.3.0.jar -lib ../ant_build/lib/jsoup-1.7.2.jar
```

Comment:
If anyone ever upgrades these files' versions with intention of replacing the files mentioned here, we need to be sure that their test pipelines run the command with those new files.